readthedocs/proxito/views/serve.py

```diff
@@ -97,7 +97,7 @@ def get(self,
         if path[-1] == '/':
             path += 'index.html'
 
-        return self._serve_docs(request, final_project=final_project, path=path)
+        return self._serve_docs(request, final_project, version_slug, path=path)
```

Comment:
I like using kwargs everywhere, if possible. I think it reduces issues downstream.